frontend/components/Filters/DoubleSelect/index.tsx

```diff
@@ -0,0 +1,177 @@
+import {
+  Fragment,
+  Key,
+  ReactChild,
+  ReactFragment,
+  ReactPortal,
+  useState,
+} from "react";
+import { Listbox, Transition } from "@headlessui/react";
+import { CheckIcon, SelectorIcon } from "@heroicons/react/solid";
+import { Option } from "types/filters";
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(" ");
+}
+
+export default function FilterDoubleSelect({
+  labelName,
+  selectOptions,
+}: {
+  labelName: string;
+  selectOptions: { id: number; name: string }[];
+}) {
+  const [selectedFirst, setSelectedFirst] = useState(selectOptions[0]);
+  const [selectedSecond, setSelectedSecond] = useState(
+    selectOptions[selectOptions.length - 1]
+  );
+
+  return (
+    <div className={"mb-3 min-w-full"}>
+      <span className="block text-sm font-medium text-gray-700">
+        {labelName}
+      </span>
+      <div className={"flex"}>
+        <Listbox value={selectedFirst} onChange={setSelectedFirst}>
+          {({ open }) => (
+            <>
+              <div className="mt-1 relative min-w-doubleFiltersSize">
```

Comment:
Since this node's parent is a flex, can't this just fill its parent? I do not understand why you needed to set min-width property on it in order to achieve full width.